scripts/akv/key-import.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+akv-key-import() {
+    set -e
+
+    AKV_NAME=${AKV_NAME:-$1}
+    if [ -z "$AKV_NAME" ]; then
+        read -p "Enter AKV name: " AKV_NAME
+    fi
+    export AKV_NAME
+
+    AKV_KEY_NAME=${AKV_KEY_NAME:-$1}
+    if [ -z "$AKV_KEY_NAME" ]; then
+        read -p "Enter AKV Key name: " AKV_KEY_NAME
+    fi
+    export AKV_KEY_NAME
+
+    az keyvault key import \
```

Comment:
why do we need to import the private key? The best practice is that the private keys is not exportable.
The signature will be generated on AKV.

Reply:
If we generate credentials on the AKV itself we need a separate member in the CCF network as credentials for the initial member is generated locally.

While the use case of a new member using credentials which never leave AKV is a great use case, it's a future step.

Reply:
https://github.com/microsoft/azure-privacy-sandbox-kms/issues/235